tests/test_remesh.py

```diff
@@ -142,24 +174,27 @@ def fixture_gyroid_mesh() -> BoxMesh:
     ],
 )
 def test_given_periodic_mesh_remesh_keeping_periodicity_for_fem_must_maintain_periodicity(
-    shape: BoxMesh,
-    request,
-    tmp_mesh_filename: str,
-    tmp_output_mesh_filename: str,
+    shape: str,
+    request: FixtureRequest,
 ) -> None:
+    # Arrange
+    input_mesh = request.getfixturevalue(shape)
+    # Act
     if USE_MMG:
-        # Arrange
+        remeshed_shape = remesh_keeping_periodicity_for_fem(input_mesh, hgrad=1.05)

```

Comment:
Maybe we should put this magic number `hgrad=1.05` in a explicit variable name

Reply:
Magic number now in `edge_length_gradient` variable